earn/src/data/constants/Values.ts

```diff
@@ -27,4 +27,5 @@ export const ALOE_II_MAX_LEVERAGE = 200;
 export const API_PRICE_RELAY_LATEST_URL = 'https://api-price.aloe.capital/price-relay/v1/latest';
 export const API_PRICE_RELAY_HISTORICAL_URL = 'https://api-price.aloe.capital/price-relay/v1/historical';
 export const API_PRICE_RELAY_CONSOLIDATED_URL = 'https://api-price.aloe.capital/price-relay/v1/consolidated';
+export const API_REDEEM_REWARD_URL = 'https://claim.aloe.capital/v1/claim';
```

Comment:
This will be `https://api-claim.aloe.capital/...` since `claim.aloe.capital` is being forwarded to Earn's page